lib/mixpanel-node.d.ts

```diff
@@ -0,0 +1,159 @@
+declare const mixpanel: mixpanel.Mixpanel;
+
+declare namespace mixpanel {
+  export type DistinctId = string;
```

Comment:
Rather than re-typing this why not just leave the definitions as `string` for simplicity purposes ?